src/coreclr/nativeaot/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/ExecutionEnvironmentImplementation.ManifestResources.cs

```diff
@@ -144,7 +144,7 @@ private static LowLevelDictionary<string, LowLevelList<ResourceInfo>> ExtractedR
         /// The dictionary's key is a Fusion-style assembly name.
         /// The dictionary's value is a list of (resourcename,index) tuples.
         /// </summary>
-        private static volatile LowLevelDictionary<string, LowLevelList<ResourceInfo>> s_extractedResourceDictionary;
+        private static volatile LowLevelDictionary<string, List<ResourceInfo>> s_extractedResourceDictionary;
```

Comment:
In avialonia-linux, all interfaces on the `List<ResourceInfo>` are preserved. This is causing the majority of size difference.